contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -170,21 +170,17 @@ contract RyskStrategy is BaseStrategy {
             pendingWithdrawal.epoch != currentWithdrawalEpoch
         ) revert RyskPendingWithdrawalNotCompleted();
 
-        uint256 currentPricePerShare = ryskLqPool.withdrawalEpochPricePerShare(
-            currentWithdrawalEpoch
-        );
-
-        uint256 sharesToWithdraw = _underlyingToShares(
+        uint256 sharesToWithdraw = _calculateSharesNeededToWithdraw(
             _amount,
-            currentPricePerShare
+            ryskLqPool.withdrawalEpochPricePerShare(currentWithdrawalEpoch)
         );
 
         _updateCachedPendingWithdrawalReceipt(
```

Comment:
The name o this function is a bit weird. I don't understand why it's "cached" or "receipt". Wouldn't `_updatePendingWithdrawal` be more than enough?

Reply:
this function isn't present anymore since we are not caching the withdrawal receipt because of the increased gas costs